app/_locales/en/messages.json

```diff
@@ -942,6 +942,12 @@
   "enableAutoDetect": {
     "message": " Enable Autodetect"
   },
+  "enableEIP1559V2": {
+    "message": "Enable EIP-1559 V2"
```

Comment:
Settings should have descriptions that users can understand. I don't see how a user would know what "EIP-1559 V2" is.

Reply:
Yep I am waiting for Jake to share details about this

Reply:
Gotcha. Could you move this into draft until then, to indicate that it's not ready for a final review?

Reply:
Message is updated now.

Reply:
Should this particular message (preference title) be something a bit more user friendly? Like `Enable Enhanced Gas Fee UI`?

Reply:
@jpuri @Gudahtt ☝️

Reply:
Sounds good let me update this.